components/MultiInsert/MultiInsert.tsx

```diff
@@ -0,0 +1,329 @@
+    switch (inputField) {
+      case 'name':
+        setWebsites(
+          websites.map((website) =>
+            website.id === id ? { ...website, nameError: status } : website,
+          ),
+        );
+        break;
+      case 'address':
+        setWebsites(
+          websites.map((website) =>
+            website.id === id ? { ...website, addressError: status } : website,
+          ),
+        );
+        break;
+      default:
+        break;
+    }
+  };
+
+  const checkWebsitesValidity = () => {
+    setFormInvalid(
+      websites.some(
+        (website) =>
+          website.nameInputRef.current?.value === '' ||
+          website.addressInputRef.current?.value === '',
+      ),
+    );
+  };
+
+  const flashToast = (toastName: string, id: number) => {
+    setToasts((current) => [...current, { text: toastName, id: id }]);
+    setTimeout(() => {
+      setToasts((current) => {
+        return current.filter((t) => t.id !== id);
+      });
+    }, 5000);
+  };
+
+  const save = () => {
+    // Remove possible empty website entries
+    const validWebsites = websites.filter(
+      (website) =>
+        website.nameInputRef.current?.value !== '' ||
+        website.addressInputRef.current?.value !== '' ||
+        website.descriptionInputRef.current?.value !== '',
+    );
+    if (validWebsites.length === 0) {
+      alert('Lisää vähintään yksi verkkosivu!');
+      return;
+    }
+
+    console.log(
+      websites.map((website) => {
+        return {
+          name: website.nameInputRef.current?.value,
+          address: website.addressInputRef.current?.value,
+          description: website.descriptionInputRef.current?.value,
+        };
+      }),
+    );
+  };
+
+  return (
+    <>
+      <div className={styles['toast-container']}>
+        {toasts.map((toast) =>
+          // Removed websites are preferably announced with their title
+          isNaN(+toast.text) ? (
+            <Block mb="s" role="alert">
+              <Toast key={toast.id}>Verkkosivu {toast.text} poistettiin</Toast>
+            </Block>
+          ) : (
+            // Websites without a title are announced more generally
+            <Block mb="s" role="alert">
+              <Toast key={toast.id}>Nimetön verkkosivu poistettiin</Toast>
+            </Block>
+          ),
+        )}
+      </div>
+      <Block variant="div">
+        <form
+          onSubmit={(event) => {
+            event.preventDefault();
+            save();
+          }}
+        >
+          <section>
+            <Block
+              className={styles['multi-insert']}
+              role="group"
+              aria-describedby="websites-heading"
+            >
+              <Label
+                asProp="h1"
+                className={styles['multi-insert_label']}
+                id="websites-heading"
+              >
+                Verkkosivu
+              </Label>
+              <Block mb="s">
+                <HintText>Voit lisätä useita verkkosivuja</HintText>
```

Comment:
Should the pattern example texts also be localized?

Reply:
Check. All texts localised now.